pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
   shared_preferences: ^2.0.8
+  rxdart: ^0.28.0
```

Comment:
sorry, but we do not want to introduce new libraries. Is it possible to achieve this with the current library?

Reply:
i could only resolve the sink issue with this library